src/main/java/org/terasology/climateConditions/HypothermiaSystem.java

```diff
@@ -48,6 +58,16 @@ public void observeDangerZone(MovedEvent event, EntityRef player) {
         }
     }
 
+    @ReceiveEvent(components = {HypothermiaComponent.class})
+    public void onHypothermia(OnAddedComponent event, EntityRef player, HypothermiaComponent hypothermia) {
+        delayManager.addPeriodicAction(player, VisibleBreathingSystem.VISIBLE_BREATH_ACTION_ID, initialDelay, breathInterval);
+    }
+
+    @ReceiveEvent(components = HypothermiaComponent.class)
+    public void beforeRemoveHypothermia(BeforeRemoveComponent event, EntityRef player, HypothermiaComponent hypothermia) {
+        delayManager.cancelPeriodicAction(player, VisibleBreathingSystem.VISIBLE_BREATH_ACTION_ID);
+    }
+
```

Comment:
On a second thought: my initial review suggested to introduce a coupling between `HypothermiaSystem` and `VisibleBreathingSystem`. Actually, there is no reason why `HypothermiaSystem` should even know that visible breath is a thing. 

Therefore, both of these methods should be moved to `VisibleBreathingSystem`, and we don't need any cross-imports anymore. 

(... still learning some best practices myself here)